packages/front-end/components/Button.tsx

```diff
@@ -70,7 +80,6 @@ const Button: FC<Props> = ({
           children
         )}
       </button>
-      {error && <pre className="text-danger ml-2">{error}</pre>}
```

Comment:
I'd prefer if we revert this change for now.  This component is used dozens of places throughout the codebase and this will likely cause some broken layouts.  Our design team is currently working on refactoring our component library, so we should fix this issue of hard-to-read errors soon.